invenio_search/api.py

```diff
@@ -65,8 +66,13 @@ def search(self, user_info=None, collection=None, **kwargs):
 
         for walker in search_walkers():
             query = query.accept(walker)
-        return Results(query)
 
+        index = cfg["SEARCH_ELASTIC_COLLECTION_INDEX_MAPPING"].get(
```

Comment:
- `cc=jobs` or `cc=hep` would solve it
- later `invenio-search` should handle non-record metadata for that case ...